custom_components/tuya_local/devices/lsc_smart_ceiling_light.yaml

```diff
@@ -0,0 +1,228 @@
+name: CCT + RGB Ceiling Light
+products:
+  - id: bwyutsrcsnwf3p5o
+    name: LSC Smart Connect Smart Ceiling Light
+
+primary_entity:
+  entity: light
+  name: Main light
+  icon: mdi:circle
+  dps:
+    - id: 63
+      type: boolean
+      name: switch
+    - id: 22
+      name: brightness
+      type: integer
+      range:
+        min: 10
+        max: 1000
+    - id: 23
+      name: color_temp
+      type: integer
+      range:
+        min: 0
+        max: 1000
+      mapping:
+        - target_range:
+            min: 2700
+            max: 6500
+
+secondary_entities:
+  - entity: light
+    name: Halo
+    icon: "mdi:circle-outline"
+    dps:
+      - id: 54
+        type: boolean
+        name: switch
+      - id: 24
+        name: rgbhsv
+        type: hex
+        format:
+          - name: h
+            bytes: 2
+            range:
+              min: 0
+              max: 360
+          - name: s
+            bytes: 2
+            range:
+              min: 0
+              max: 1000
+          - name: v
+            bytes: 2
+            range:
+              min: 0
+              max: 1000
+      - id: 24
```

Comment:
I think brightness should not be needed, as this light is only supporting color mode, so the v portion of rgbhsv above will be used for brightness.

Reply:
No, the V value is ignored by the light when changed by Home assistant and only works when sending it separately. Took me hours to figure out.

Reply:
Yes, but the code is already there to map the separate brightness onto the rgbhsv field when a light is in color mode.

Reply:
I know, but that part is ignored when changing the brightness. It doesn't affect the lights when changed in Home Assistant.